test/integration/addons_test.go

```diff
@@ -67,7 +67,7 @@ func TestAddons(t *testing.T) {
 		args = append(args, "--addons=ingress")
 	}
 	if !arm64Platform() {
-		args = append(args, "--addons=helm-tiller")
+		args = append(args, "--addons=helm-tiller", "--addons=auto-pause")
```

Comment:
Why don't we test auto-pause on arm64?

Reply:
currently we dont have auto-pause on arm yet